### ISSUE TITLE
skip-link test disabled?

### ISSUE THREAD

**Issue by badtant** [dequelabs/axe-core#208]
https://github.com/dequelabs/axe-core/blob/master/lib/rules/skip-link.json

Hi, why is this test disabled?
Same goes for region and heading-order

**Comment by marcysutton** [comment]
It looks like Best Practice rules are disabled by default so they don't cause false positives. https://github.com/dequelabs/axe-core/commit/412d536a46b5cde9e9d4d5eaa7c7f9ce8245e1dd

**Comment by badtant** [comment]
Hmm, because they are not done or what is the reason? Can I enable them with some config option? Otherwise maybe they shouldn't be in this list:
https://github.com/dequelabs/axe-core/blob/master/doc/rule-descriptions.md

**Comment by dylanb** [comment]
@badtant you can turn rules on/off using the axe.configure function. Simply call

```
axe.configure({
  rules: [{
    "skip-link": {
      "enabled": true
    }
  }]
});
```

Note, that you will have to do this call in each iframe on your page

**Comment by badtant** [comment]
Cool, thanks!
Is this also possible when using react-axe?

**Comment by dylanb** [comment]
@badtant no, it is not, but that would be a great contribution to react-axe
